inception-doc/src/main/resources/META-INF/asciidoc/user-guide.adoc

```diff
@@ -143,7 +143,7 @@ include::{include-dir}projects_layers.adoc[leveloffset=+1]
 
 include::{include-dir}projects_knowledge-base.adoc[leveloffset=+1]
 
-include::{include-dir}projects_recommendation.adoc[leveloffset=+1]
+include::{include-dir}projects_recommendation.adoc[leveloffset=+2]
```

Comment:
This is confusing because the section is at the same level as the KB documentation. Better leave it at +1.

Reply:
I would rather change the kb documentation to +2, because it starts at `==` inside.

Reply:
The main file (containing the includes) is the basis. Sub-sections should be included as +1. If a sub-section has additional sub-sections, these should be included as +2. KB and Recommender are both sub-sections of the main file, so they should be +1.

Reply:
I think KB needs to be at 2, as it is `Main->Projects->Knowledge Bases` and `Main->Projects->Recommender`

Reply:
Hm, right - I missed the prefix. Then let's change the KB to +2.